backend/src/schema/resolvers/socialMedia.js

```diff
@@ -26,5 +23,21 @@ export default {
 
       return socialMedia
     },
+    UpdateSocialMedia: async (object, params, context, resolveInfo) => {
+      const session = context.driver.session()
+      await session.run(
+        `MATCH (owner: User { id: $userId })-[:OWNED]->(socialMedia: SocialMedia { id: $socialMediaId })
+        SET socialMedia.url = $socialMediaUrl
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-18T07:01:18Z" title="Thursday, July 18th 2019, 9:01:18 am +02:00">Jul 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -26,5 +23,22 @@ export default {
 
       return socialMedia
     },
+    UpdateSocialMedia: async (object, params, context, resolveInfo) => {
+      const session = context.driver.session()
+      await session.run(
+        `MATCH (owner:User {id: $userId}), (socialMedia:SocialMedia {id: $socialMediaId})
+        WHERE (socialMedia)<-[:OWNED]-(owner)
```


Both lines can be expressed in one as:
`MATCH (owner: User { id: $userId })-[:OWNED]->(socialMedia: SocialMedia { id: $socialMediaId })`

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-07-23T13:15:01Z" title="Tuesday, July 23rd 2019, 3:15:01 pm +02:00">Jul 23, 2019</time>_

---


Thank you! I knew there had to be a better solution. :smile: